### ISSUE TITLE
Unable to add/update packages on repman.io

### ISSUE THREAD

**Issue by dear-leader** [repman-io/repman#611]
Updates to packages in Github are not being reflected in the repman.io cloud service. I can see that webhooks are being received, and have also tried force updating to no avail.

Attempting to add a new package doesn't work either, stuck in a state of "synchronizing from ...".

The behavior seems to be the same as discussed in issues https://github.com/repman-io/repman/issues/565 and https://github.com/repman-io/repman/issues/566, but not sure the cause. Any ideas?

**Comment by juz501** [comment]
I'm getting similar issues today.

**Comment by cstrolink** [comment]
Same here

**Comment by Gallmond** [comment]
Similar issues since yesterday evening.

**Comment by erkenes** [comment]
> It works again :)

No, some packages are still not found although they are stored in the registry and are displayed

**Comment by juz501** [comment]
> > It works again :)
> 
> No, some packages are still not found although they are stored in the registry and are displayed in the webview.
> 
> Edit: If you press "update" in the webview on those packages they are available again

Yea, we have the same issue.. however, Update button isn't working for us. (well it looks like it works but never actually fixes it)

**Comment by erkenes** [comment]
> > > It works again :)
> > 
> > 
> > No, some packages are still not found although they are stored in the registry and are displayed in the webview.
> > Edit: If you press "update" in the webview on those packages they are available again
> 
> Yea, we have the same issue.. however, Update button isn't working for us. (well it looks like it works but never actually fixes it)
> 
> Edit: the 'update' button on the list page worked, but the 'update' button on the package details page did not work.
> 
> Edit 2: looks like it just took a real long time to update.. not sure if it's the various buttons.

You have to use the update button on the detail page of every single package. The button in the list doesn't work well and takes a long time to update.
If I press the button on the detail page then the package is direct available again.

**Comment by hugronaphor** [comment]
This suddenly broke for me a few days ago and triggering update didn't help
